qiskit/chemistry/components/variational_forms/uccsd.py

```diff
@@ -124,9 +146,20 @@ def __init__(self, num_qubits, depth, num_orbitals, num_particles,
                                           sq_paulis, sq_list, tapering_values, and cliffords
             shallow_circuit_concat (bool): indicate whether to use shallow (cheap) mode for
                                            circuit concatenation
+            method_singles (str): specify the single excitation considered. 'alpha', 'beta',
+                                  'both' only alpha or beta spin-orbital single excitations or
+                                  both (all of them)
+            method_doubles (str): specify the single excitation considered. 'ucc' (conventional
```

Comment:
This seems to be missing the 'pucc' mode

Reply:
Thanks, also corrected.